email_api/send_email.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+nodeMailer = require('nodemailer');
+
+//function to send the email
+function emailData(mail_Obj) {
```

Comment:
mail_obj should be camel case

Reply:
sir i have changed it to camel case as "mail_Obj"